pageflow.gemspec

```diff
@@ -161,4 +161,7 @@ Gem::Specification.new do |s|
 
   # Semantic versioning rake tasks
   s.add_development_dependency 'semmy', '~> 0.2.0'
+
+  # A gem that makes it easy to write specs for your Rails 3 Generators.
+  s.add_development_dependency 'ammeter', '~> 1.1.3'
```

Comment:
For versions `>= 1.0` I tend to only depend on the minor, i.e. `~> 1.1`, assuming Semver is being followed. Or is there a reason to stay on `1.1` stable?